### PR TITLE
remove 'istiotesting' parent section for 'onenamespace' values.

### PR DESCRIPTION
**Pull request by morvencao**
'istiotesting' has been deprecated, remove this section in `values-istio-one-namespace-trust-domain.yaml`

### CODE REVIEW

File: install/kubernetes/helm/istio/values-istio-one-namespace-trust-domain.yaml [hunk 1]
```diff
@@ -14,5 +14,4 @@ global:
 
   trustDomain: test.local
 
-istiotesting:
   oneNameSpace: true
```
Comment (ymesika) [inline install/kubernetes/helm/istio/values-istio-one-namespace-trust-domain.yaml]: Shouldn't you remove this line too?

Comment (morvencao) [reply install/kubernetes/helm/istio/values-istio-one-namespace-trust-domain.yaml]: We should use this one: https://github.com/istio/istio/blob/master/install/kubernetes/helm/istio/values.yaml#L278

Comment (ymesika) [reply install/kubernetes/helm/istio/values-istio-one-namespace-trust-domain.yaml]: Ok, that's why my search couldn't find it..

**Comment by linsun** [comment]
@wenchenglu can we get this merged for 1.1?

**Pull request merged**